doc/alertmanager-mixin/alerts.libsonnet

```
@@ -44,7 +44,7 @@
               (
                 rate(alertmanager_notifications_failed_total{%(alertmanagerSelector)s}[5m])
               /
-                rate(alertmanager_notifications_total{%(alertmanagerSelector)s}[5m])
+                on (integration) group_left rate(alertmanager_notifications_total{%(alertmanagerSelector)s}[5m])
```

I'd suggest to ignore the `reason` label since it's the only one that's different between the 2 metrics. It will change a bit the alert condition in case of multiple reasons (e.g. the sum for all reasons might be above the threshold while the individual reasons are below). But the current threshold value is low enough that it shouldn't be an issue in practice.

```suggestion
                ignoring (reason) group_left rate(alertmanager_notifications_total{%(alertmanagerSelector)s}[5m])
```

---

Thanks for your suggestion. Fixed.